### PR TITLE
tags/admin/*.tag: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
All files in `tags/admin/*.tag`

### CODE REVIEW

**Comment by sujeet14108** [comment]
@whipermr5 why this is failing ?

**Comment by whipermr5** [comment]
When the build fails on master, no coverage report is generated, resulting in the error. It should be fine now.

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/admin/activity/activityLogTableRow.tag [hunk 1]
```diff
@@ -4,80 +4,80 @@
...
-                <input class="ifShowTestData_for_person" type="hidden" name="testdata"
-                    value="false">
-            </h4>
-            <div class="log-message">${log.message}</div>
-        </form>
-    </td>
+        <small>
+          id: ${log.logId}
+          [
+          ${log.userName}
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/activity/activityLogTableRow.tag]: The above three lines can be combined.

File: src/main/webapp/WEB-INF/tags/admin/email/compose.tag [hunk 2]
```diff
@@ -5,69 +5,69 @@
...
+    </p>
+    <p>
+      <button type="button" id="composeSubmitButton">Send</button>
+      <button type="button" id="composeSaveButton">save</button>
+    </p>
+  </form>
+
+  <div id="adminEmailGroupReceiverListUploadBox" style="display:none;">
+    <form id="adminEmailReceiverListForm" action="" method="POST" enctype="multipart/form-data">
+      <span id="adminEmailGroupReceiverListInput"> <input
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/email/compose.tag]: Move the `input` tag to the next line.

File: src/main/webapp/WEB-INF/tags/admin/email/compose.tag [hunk 3]
```diff
@@ -5,69 +5,69 @@
...
-            <span id="adminEmailFileInput"> <input
-                type="file"
-                name="<%=Const.ParamsNames.ADMIN_EMAIL_IMAGE_TO_UPLOAD%>"
-                id="adminEmailFile">
-            </span>
-        </form>
+  <div style="display: none;">
+    <form id="adminEmailFileForm" action=""
+        method="POST" enctype="multipart/form-data">
+      <span id="adminEmailFileInput"> <input
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/email/compose.tag]: Move the `input` tag to the next line.

File: src/main/webapp/WEB-INF/tags/admin/email/log/emailLogTableRow.tag [hunk 4]
```diff
@@ -4,29 +4,29 @@
...
-        </ul>
-    </div>
+  <div class="well well-sm">
+    <ul class="list-group">
+      <li class="list-group-item list-group-item-success emailLog-text email-content">
+        <small>${log.unsanitizedContent}</small>
+      </li>
+    </ul>
+  </div>
    </td>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/email/log/emailLogTableRow.tag]: Remove one space from this.

File: src/main/webapp/WEB-INF/tags/admin/email/log/emailLogTableRow.tag [hunk 5]
```diff
@@ -4,29 +4,29 @@
...
 <tr class="email-log-content-unsanitized" style="display:none;">
   <td colspan="3">
-    <div class="well well-sm">
-        <ul class="list-group">
-            <li class="list-group-item list-group-item-success emailLog-text email-content">
-                <small>${log.unsanitizedContent}</small>
-            </li>
-        </ul>
-    </div>
+  <div class="well well-sm">
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/email/log/emailLogTableRow.tag]: Add two spaces to this entire `div` block.

**Comment by sujeet14108** [comment]
Done :)

**Comment by sujeet14108** [comment]
@wkurniawan07 Ready for review :)

File: src/main/webapp/WEB-INF/tags/admin/sessions/filter.tag [hunk 6]
```diff
@@ -3,122 +3,117 @@
...
+                </c:forEach>
+              </select>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-4 border-left-gray">
+          <div class="row">
+            <div class="col-md-12">
+              <label for="<%=Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE%>" class="control-label">
+                Time Zone</label>
```
Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/admin/sessions/filter.tag]: Done

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag [hunk 7]
```diff
@@ -9,216 +9,216 @@
...
+            </div>
+          </div>
+        </small>
+      </div>
+
+    </div>
+
+    <input type="hidden" name="pageChange" value="true">
+
+    <%-- This parameter determines whether the logs with requests contained in "excludedLogRequestURIs"
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: If I am not mistaken, the convention for multiline JSP comment is:
```jsp
<%--
  - This is the first line.
  - This is the second line.
  --%>
```
Update this and all other multiline comments below (4 instances) to follow that convention.

File: src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag [hunk 8]
```diff
@@ -9,216 +9,216 @@
...
+    in AdminActivityLogPageData should be shown. Use "?all=true" in URL to show all logs. This will keep showing all
+    logs despite any action or change in the page unless the page is reloaded with "?all=false"
+    or simply reloaded with this parameter omitted. --%>
+    <input type="hidden" name="all" value="${shouldShowAllLogs}">
+
+    <%-- This determines whether the logs related to testing data should be shown. Use "testdata=true" in URL
+    to show all testing logs. This will keep showing all logs from testing data despite any action or change in the page
+    unless the page is reloaded with "?testdata=false"  or simply reloaded with this parameter omitted. --%>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: In addition to the multiline comment convention, remove the extra whitespace between `"?testdata=false"` and `or`.

File: src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag [hunk 9]
```diff
@@ -9,216 +9,216 @@
...
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: Similarly here for the extra whitespace.

File: src/main/webapp/WEB-INF/tags/admin/email/log/filterPanel.tag [hunk 10]
```diff
@@ -5,107 +5,97 @@
...
-                                <strong>
-                                    Possible Labels:</strong>&nbsp;after,
-                                    before, receiver, subject,
-                                    info, version<br>
-                                <ul>
+          <div class="form-group">
+            <div class="col-md-12">
+              <div class="form-control-static">
+                <strong>Possible Labels:</strong>
+                  &nbsp;after, before, receiver, subject, info, version<br>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/email/log/filterPanel.tag]: Remove 2 spaces here.

File: src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag [hunk 11]
```diff
@@ -9,216 +9,226 @@
...
+    <%--
+      - This determines whether the logs related to testing data should be shown. Use "testdata=true" in URL
+      - to show all testing logs. This will keep showing all logs from testing data despite any action or change in the page
+      - unless the page is reloaded with "?testdata=false" or simply reloaded with this parameter omitted.
+      --%>
+    <input type="hidden" name="testdata" value="${shouldShowTestData}">
+  </form>
+
+  <%--
+    - This form is used to store parameters for ajaxloader only
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: This is just one line. It does not need to be displayed with `<%-- ... - ... --%>`.

Comment (sujeet14108) [reply src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag]: Done

**Pull request merged**